### PR TITLE
wholeBodyDyanamics: Disable IMU when Fixed frame kinematics source is used

### PR DESCRIPTION
**Pull request by prashanthr05**
This features disables the check for attaching IMU sensors while opening the device.

### CODE REVIEW

File: devices/wholeBodyDynamics/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ For an overview on `wholeBodyDynamics` and to understand how to run the device,
 |:--------------:|:--------------:|:-----------------:|:-----:|:-------------:|:--------:|:-----------------------------------------------------------------:|:-----:|
 | axesNames      |      -         | vector of strings |   -   |   -           | Yes      | Ordered list of the axes that are part of the remapped device.    |       |
 | modelFile      |      -         | path to file      |   -   | model.urdf    | No       | Path to the URDF file used for the kinematic and dynamic model.   |       |
-| assume_fixed    |                | frame name        |   -   |     -         | No       | If it is present, the initial kinematic source used for estimation will be that specified frame is fixed, and its gravity is specified by fixedFrameGravity. Otherwise, the default IMU will be used. | |
+| assume_fixed    |                | frame name        |   -   |     -         | No       | If it is present, the initial kinematic source used for estimation will be that specified frame is fixed, and its gravity is specified by fixedFrameGravity. The IMU related parameters are overriden if this parameter exists. Otherwise, the default IMU will be used. | |
```
Comment (traversaro) [inline devices/wholeBodyDynamics/README.md:17]: Are overriden in which sense? With which values are overriden?

File: devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp [hunk 2]
```diff
@@ -1658,7 +1661,11 @@ bool WholeBodyDynamicsDevice::attachAll(const PolyDriverList& p)
     ok = ok && this->attachAllControlBoard(p);
     ok = ok && this->attachAllVirtualAnalogSensor(p);
     ok = ok && this->attachAllFTs(p);
-    ok = ok && this->attachAllIMUs(p);
+    
+    if (settings.kinematicSource == IMU)
```
Comment (traversaro) [inline devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp:1665]: If I remember correctly the settings can also be changed via RPC. It probably could be worth to check the other places in which `settings.kinematicSource` is checked, and raise an error if kinematicsSource is switched to IMU after a start without IMU is done.

File: devices/wholeBodyDynamics/README.md (hunk 1 above)
Comment (prashanthr05) [reply devices/wholeBodyDynamics/README.md:17]: Overridden in the sense, those settings from the configuration file related to the IMU would never be used.

File: devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp (hunk 2 above)
Comment (prashanthr05) [reply devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp:1665]: You remember correctly. I agree. I will handle that as well.

Comment (prashanthr05) [reply devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp:1666]: Fixed in https://github.com/robotology/whole-body-estimators/pull/95/commits/f6c3a8122a6a3aaa4582f2faf8c5ec13bf54c5a2

File: devices/wholeBodyDynamics/README.md (hunk 1 above)
Comment (prashanthr05) [reply devices/wholeBodyDynamics/README.md]: Fixed in https://github.com/robotology/whole-body-estimators/pull/95/commits/f6c3a8122a6a3aaa4582f2faf8c5ec13bf54c5a2

**Review by traversaro — APPROVED** [review]

**Comment by prashanthr05** [comment]
Merged, thanks!

**Pull request merged**